caddyfile.go

```diff
@@ -356,9 +352,12 @@ func parseCaddyfile(h httpcaddyfile.Helper) (*authz.Authorizer, error) {
 							}
 						}
 						p.LoginHintValidators = validationArguments
-					} else {
+						break
+					default:
 						p.LoginHintValidators = []string{"alphanumeric"}
+						break
```

Comment:
Let’s the default be all three validations. Any of them passes, it is good to go.